### PR TITLE
use index.auto_expand_replicas '0-1' by default

### LINKED ISSUE #414: [Index Template] Use index.auto_expand_replicas '0-1'
The APM index / indices should consider using the `index.auto_expand_replicas` setting, set to `"0-1"` in favor of `index.number_of_replicas` set to `1` (or leaving it as the default).

Doing so will allow single node (e.g., dev) clusters to startup as green, which gives a better getting started experience.

### PR DESCRIPTION
**Pull request by graphaelli**
closes #414

### CODE REVIEW

**Comment by graphaelli** [comment]
Waiting for input from beats team before merging.

**Comment by graphaelli** [comment]
Forgot to update reference config too

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -22,6 +22,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485] 
 
 ==== Added
+- Use index.auto_expand_replicas '0-1' by default {pull}503[503]
```
Comment (roncohen) [inline CHANGELOG.asciidoc]: nitpick: Should be at the bottom of the section

Comment (graphaelli) [reply CHANGELOG.asciidoc]: 🤦‍♂️

**Comment by graphaelli** [comment]
@roncohen besides that change, are we ok with this default?

**Comment by graphaelli** [comment]
reference config update will come as part of #496 for 6.x if we merge this in.  No solution currently to doc this on master but will work on that in follow on issue

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
Holding off on this for now.  There is concern that users will not realize that they have a single replica by default, and that is a bigger risk than making single node clusters yellow.  Discussions to continue on this topic soon.

**Comment by roncohen** [comment]
makes sense 👍

**Comment by graphaelli** [comment]
closing for now so we don't accidentally merge

**Pull request closed without merging**